pam_authz/docker/pam-builder.dockerfile

```diff
@@ -1,9 +1,20 @@
 FROM golang:1.8
 
-RUN apt-get update && apt-get install -y \
-    libpam0g-dev && \
+RUN apt-get update && \
+	apt-get install -y --no-install-recommends \
+    libpam0g-dev \
+    apt-utils \
+    libcurl4-gnutls-dev && \
     rm -rf /var/lib/apt/lists/*
 
+RUN wget http://www.digip.org/jansson/releases/jansson-2.11.tar.gz && \
```

Comment:
Would be nice to see a comment indicating this is a json lib. The lib is also on GitHub (https://github.com/akheron/jansson), which seems like a more reliable place to obtain it from

Reply:
Added comment, but the Github version seems to have autoconf problems.